plugin-gradle/src/main/java/com/diffplug/gradle/spotless/ShellExtension.java

```diff
@@ -23,7 +23,7 @@
 import com.diffplug.spotless.shell.ShfmtStep;
 
 public class ShellExtension extends FormatExtension {
-	private static final String SHELL_FILE_EXTENSION = "*.sh";
+	private static final String SHELL_FILE_EXTENSION = "**/*.sh";
```

Comment:
It's likely the case that someone isn't always going to be storing their shell scripts as siblings to the build file.